api/FocusEvent.json

```diff
@@ -6,10 +6,10 @@
         "spec_url": "https://w3c.github.io/uievents/#interface-focusevent",
         "support": {
           "chrome": {
-            "version_added": true
+            "version_added": "26"
```

Comment:
Confirmed by testing https://staging-dot-mdn-bcd-collector.appspot.com/tests/api/FocusEvent in Chrome 25 and 26. (The test has been updated since this PR was created.)